QUEST_LV_0100.tsv

```diff
@@ -1,124 +1,124 @@
-QUEST_LV_0100_20150317_000113	I told her not to get excited even if she finds something.{nl}She did it again.
-QUEST_LV_0100_20150317_000114	Thanks.{nl}I will go look for Irene.
-QUEST_LV_0100_20150317_000115	Technician Heinen
-QUEST_LV_0100_20150317_000116	{memo X}
-QUEST_LV_0100_20150317_000117	{memo X}
-QUEST_LV_0100_20150317_000118	{memo X}
-QUEST_LV_0100_20150317_000119	{memo X}Much better than I expected.{nl}I want to propose a deal to you.
-QUEST_LV_0100_20150317_000120	{memo X}
-QUEST_LV_0100_20150317_000121	{memo X}
+QUEST_LV_0100_20150317_000001	$Sentinel
+QUEST_LV_0100_20150317_000002	$If you choose to go to Klaipeda, I recommend taking the Shadow Forest Road.{nl}The deep woods are dangerous, so try to stay away from there if you can.
+QUEST_LV_0100_20150317_000003	$Man in Hiding
+QUEST_LV_0100_20150317_000004	$Fi... Finally! Has help arrived?{nl}Huh?! It's just a kid? Hey... Are there still Vubbes outside?
+QUEST_LV_0100_20150317_000005	$Ugh. Go away! Don't talk to me! We'll get caught by the Vubbes!{nl}This is my place so leave!
+QUEST_LV_0100_20150317_000006	$What? Don't lie to me! I can smell the Vubbes from here.{nl}You can't fool an adult! Go away! Do not come near this barrel!
+QUEST_LV_0100_20150317_000007	$Fi... Finally! Has help arrived?{nl}Huh?! It's just a kid? What are you looking at? Close the lid and move along!{nl}
+QUEST_LV_0100_20150317_000008	$What's that paper you're holding? It looks familiar...{nl}Let me see that! What are you doing? Hey, there's an adult speaking to you!
+QUEST_LV_0100_20150317_000009	$This paper, it's mine! I'm never giving it away again!!{nl}It's.. not something you need - I'll just rip it apart. There.
+QUEST_LV_0100_20150317_000010	$Sheesh, that's lame...
+QUEST_LV_0100_20150317_000011	$This isn't the first sign to appear after all the Goddesses vanished.{nl}Maybe one of these people will be the one to bring the Goddesses back.
+QUEST_LV_0100_20150317_000012	$Soldier Block
+QUEST_LV_0100_20150317_000013	$Be warned!{nl}Monsters are attacking, so all civilians must evacuate!{nl}Why are those civilians in the military zone anyway?{nl}
+QUEST_LV_0100_20150317_000014	$Ah, so you are the Revelator! I'm sorry, I didn't recognize you.{nl}Monsters are attacking the supply base at the moment.{nl}I need to get some important rations out from the base. Please help me!
+QUEST_LV_0100_20150317_000015	$It was peaceful not too long ago, but now they are coming in groups.{nl}What's happening exactly?
+QUEST_LV_0100_20150317_000016	$Even so, I felt uneasy...{nl}Who knew that they'd attack in such an absurd manner?
+QUEST_LV_0100_20150317_000017	$The rations are in the First Supply Warehouse, which you can get to with the road on the upper side.{nl}Bring however little leftovers you can find... 
+QUEST_LV_0100_20150317_000018	$I wrote down who Sparnas is, maybe it will be useful in the future.{nl}Though I may not appear so right now, I was a biologist four years ago.
```

Comment:
"I wrote down who Sparnas is, maybe it will be useful in the future." -> "I wrote down who Sparnas is. Maybe it will be useful in the future."

They're two separate sentences.